### ISSUE TITLE
Not all local branches can merge with its remote-tracking branch by a single git pull command at once

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#533]
book\02-git-basics\sections\remotes.asc line 192

The merge feature of git pull only works on current branch. So, 

```
[...] and multiple branches that are automatically merged when you run `git pull`.
```

could be?

```
[...] and multiple branches that can be merged automatically when you run `git pull`.
```

**Comment by ben** [comment]
Yeah, that's better.

**Comment by ben** [comment]
Fixed with #536.
